datastream-server-api/src/test/java/com/linkedin/datastream/server/TestFlushlessEventProducerHandler.java

```diff
@@ -32,11 +35,18 @@
   private static final String TOPIC = "MyTopic";
   private static final Random RANDOM = new Random();
 
-  @Test
-  public void testSingleRecord() {
-    RandomEventProducer eventProducer = new RandomEventProducer();
-    FlushlessEventProducerHandler<Long> handler = new FlushlessEventProducerHandler<>(eventProducer);
+  private static final CallbackStatusFactory<Long> OFFSET_CHECKPOINT_TRACKING_STRATEGY_WITH_NON_COMPARABLE_OFFSETS =
+      ReflectionUtils.createInstance(CallbackStatusWithNonComparableOffsetsFactory.class.getName());
+
+  private static final CallbackStatusFactory<Long> OFFSET_CHECKPOINT_TRACKING_STRATEGY_WITH_COMPARABLE_OFFSETS =
+      ReflectionUtils.createInstance(CallbackStatusWithComparableOffsetsFactory.class.getName());
 
+  /**
+   * Helper function for testing single record send for both comparable and non comparable offsets
+   * @param eventProducer
+   * @param handler
```

Comment:
incomplete javadocs.

Reply:
do you need these functions to be public?

Reply:
same comment applies to couple of other places in this file. Can you please fix them as well.